common/perception_utils/test/src/test_perception_utils.cpp

```diff
@@ -375,6 +375,68 @@ TEST(perception_utils, test_get2dIoU)
   }
 }
 
+TEST(perception_utils, test_get2dGeneralizedIoU)
```

Comment:
Any reason not to have unit test for getMaxMin2d?
Or you can move getMaxMin2d to anonymous namespace so that other file cannot refer to it.

Reply:
reflect the indication in https://github.com/autowarefoundation/autoware.universe/pull/1902/commits/975e98db152e6229c8f8f3cb674e93f98ff007ee